clients/src/main/resources/common/message/DescribeConfigsResponse.json

```diff
@@ -59,7 +59,11 @@
             "about": "The synonym value." },
           { "name": "Source", "type": "int8", "versions": "1+",
             "about": "The synonym source." }
-        ]}
+        ]},
+        { "name": "ConfigValueType", "type": "int8", "versions": "3+", "default": "0", "ignorable": true,
```

Comment:
Any reason this is called `ConfigValueType` here while it is called `ConfigType` everywhere else?

Reply:
No, I should have used `ConfigType` everywhere.